### PR TITLE
[UserNotifications] Fix xtro test results

### PR DESCRIPTION
**Pull request by dalexsoto**
* ActionIdentifier from UNNotificationResponse class can be
  a NSString defined in UNNotificationActionIdentifier but can
  also be a custom defined identifier so we can't really strong type it
* Renamed enums to use the same name as defined in the header because
  we don't really know how this framework will evolve over time
  and there is a chance that the current naming can create confusion
  in the long run so we play safe here

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build failure

File: tests/xtro-sharpie/ios.pending [hunk 1]
```diff
@@ -46,6 +46,16 @@
...
+# UserNotifications
+
+## Shorter enum names
+!missing-enum! UNNotificationContentExtensionMediaPlayPauseButtonType not bound
+!missing-enum! UNNotificationContentExtensionResponseOption not bound
+!unknown-native-enum! UNResponseOption bound
+!unknown-native-enum! UNMediaPlayPauseButtonType bound
+
+## Field to Enum manual mapping
+!unknown-native-enum! UNActionIdentifier bound
```
Comment (spouliot) [inline tests/xtro-sharpie/ios.pending]: if it's a Xamarin helper enum then it should not be `[Native]` (no need) and that should remove this entry from xtro results

File: tests/xtro-sharpie/watchos.pending [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # TO BE FIXED - but maybe only in the far future / profiles
 
+# UserNotifications
+
+## Field to Enum manual mapping
+!unknown-native-enum! UNActionIdentifier bound
```
Comment (spouliot) [inline tests/xtro-sharpie/watchos.pending]: same

File: tests/xtro-sharpie/ios.pending [hunk 3]
```diff
@@ -46,6 +46,16 @@
 ## fixed for XAMCORE_4_0
 !incorrect-protocol-member! QLPreviewItem::previewItemTitle is OPTIONAL and should NOT be abstract
 
+# UserNotifications
+
+## Shorter enum names
```
Comment (spouliot) [inline tests/xtro-sharpie/ios.pending]: not sure about those... we don't know the future of that framework and they could become confusing in the future (even if right now I like them better).

in any case a type rename should be added in the comments of the binding files (to ease audits) and this should also move to the `Helpers.cs` file

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build failure

**Comment by dalexsoto** [comment]
build

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**